dataminer/Functions/Video_Thumbnails/Configuring_Video_Thumbnails/Configuring_a_video_thumbnail_in_a_web_component.md

```diff
@@ -0,0 +1,23 @@
+---
+uid: Configuring_a_video_thumbnail_in_a_web_component
+---
+
+# Configuring a video thumbnail in a web component
+
+When you have added a [web component](xref:DashboardWeb) in a dashboard or low-code app, you can configure it as follows to show a video thumbnail:
+
+1. Go to the *Settings* pane.
+
+1. In the *URL* box, enter the video URL. You can use IntelliSense to insert the DMA placeholder `{URL.DMAIP.Value}`. For example:
+
+   ```txt
+   #https://{URL.DMAIP.Value}/VideoThumbnails/Video.htm?type=HTML5&source=https://videoserver/video.mp4
+   ```
+
+   The URL consists of the following main components:
+
+   - `#`: The "#" in front of the URL ensures that the video is displayed in an embedded browser.
```

Comment:
This only applies to Visio, not here in a web component where it's always embedded